ui/inview.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @fileoverview Utility functions related to whether an element is visible in
+ * the viewport.
+ */
+
+var classes = require('../utils/classes');
+var scrolldelegator = require('../scrolldelegator');
+
+
+/**
+ * Returns whether an element is visibile in the viewport.
+ * @param {Element} el Element to check.
+ * @param {number=} opt_offset Offset to apply when checking
+ *    whether the element is in view. The value of opt_offset should
+ *    be a fraction (e.g. 0.85) and the resulting offset value is
+ *    determined by multiplying opt_offset with the element's height.
+ * @return {boolean} Whether the element is in view.
+ */
+function isElementInView(el, opt_offset) { 
```

Comment:
Do you think we should move this to `ui/index.js` so people can do `ui.isElementInView` (which is a utility func without any dependencies) -- and keep the non-utility funcs and classes in their own named files?